framework/src/st_things/things_stack/easy-setup/easysetup_manager.c

```diff
@@ -727,6 +730,58 @@ void dev_conf_prov_cb_in_app(es_dev_conf_prov_data_s *event_data)
 		// check payload
 		THINGS_LOG_D(THINGS_DEBUG, TAG, "Datetime : %s", event_data->datetime);
 	}
+
+	//Added by THINGS to set time in appliance yyyy-mm-ddThh-mm-ss
+	char chYear[5], chMonth[3], chDay[3], chHour[3], chMin[3], chSec[3];
```

Comment:
every variable should be defined at new line. That is our coding rule.

Reply:
OK Thanks.
I will fix it